modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/store.js

```diff
@@ -1,137 +1,8 @@
 AUI.add(
```

Comment:
We should delete this file. `Liferay.Store` will always be overridden from `global.es.js`, and API is the same.